radio/src/mavlink_telem_camera.cpp

```diff
@@ -0,0 +1,268 @@
+/*
+ * (c) www.olliw.eu, OlliW, OlliW42
+ */
+
+#include "opentx.h"
+
+constexpr float FPI = 3.141592653589793f;
```

Comment:
I’d suggest moving the complete telemetry code into the corresponding directory.

Reply:
I was thinking about that ... but refrained form moving it to telemetry, since - despite the saying - mavlink is actually not telemetry and really functions totally differently and hence has also very different resource usage and code layout ... i.e. architecturally it doesn't fit into that folder and I have difficulties considering it as the "corresponding directory" :)
I was thinking about a folder mavlink, but it may confuse with the thirdparty folder, or a folder mavlink_telem...
Maybe a folder /telemetry/mavlink_telem ??

Reply:
A mavlink folder inside telemetry is perfect for me.

Reply:
ok

Reply:
done

Reply:
Thx!